### DISCUSSION TITLE
我觉得目前这个前端界面……不好看

### DISCUSSION THREAD

**Discussion by 212-mei** [AlistGo/alist#233] [Ideas]
虽然我看到后台可以自定义body等，相当于样式可以自己写，但是我还是我还是想说，为啥不使用原先的那一套主题，我感觉之前就挺好看的🌝🌝🌝🌝能不能出几套主题然后后台可以选呢？(只是个人认为目前这个默认前端页面不好看，无意冒犯)而且我并没有自己写出好看的页面的能力

**Comment by xhofe** [comment]
你觉得原先的好看，我觉得现在的好看，萝卜青菜，各有所爱。
我做不到让每个人都满意，让所有人都喜欢，我也没有精力去维护多个主题，所以最好的方式就是自己写令自己满意的样式。

**Reply by Zynismus16** [reply]
个人建议，可以出个自定义前端功能，也就是静态前端可以前后端分离，这样自己改后不用合着后端一起编译，只编译前端。
emmm好像v2有这个功能？

**Reply by lhuanyun** [reply]
好看极了…

**Reply by haorwen** [reply]
> 个人建议，可以出个自定义前端功能，也就是静态前端可以前后端分离，这样自己改后不用合着后端一起编译，只编译前端。 emmm好像v2有这个功能？

早就有分离了啊，https://github.com/alist-org/alist-web
